addons/sale/models/sales_team.py

```diff
@@ -51,11 +65,74 @@ def _compute_invoiced(self):
             ])
             team.invoiced = sum(invoices.mapped('amount_untaxed_signed'))
 
+    def _graph_date_column(self):
+        if self.dashboard_graph_model in ['sales', 'invoices']:
+            return 'date'
+        return super(CrmTeam, self)._graph_date_column()
+
+    def _graph_y_query(self):
+        if self.dashboard_graph_model == 'sales':
+            return 'SUM(price_subtotal)'
+        elif self.dashboard_graph_model == 'invoices':
+            return 'SUM(price_total)'
+        return super(CrmTeam, self)._graph_y_query()
+
+    def _graph_sql_table(self):
+        if self.dashboard_graph_model == 'sales':
+            return 'sale_report'
+        elif self.dashboard_graph_model == 'invoices':
+            return 'account_invoice_report'
+        return super(CrmTeam, self)._graph_sql_table()
+
+    def _extra_sql_conditions(self):
+        if self.dashboard_graph_model == 'sales':
+            return "AND state in ('sale', 'done')"
+        elif self.dashboard_graph_model == 'invoices':
+            return "AND state = 'paid'"
+        return super(CrmTeam, self)._extra_sql_conditions()
+
+    def _graph_title_and_key(self):
+        if self.dashboard_graph_model == 'sales':
+            return [_('Sales'), _('Untaxed Total')]
+        elif self.dashboard_graph_model == 'invoices':
+            return [_('Invoices'), _('Untaxed Total')]
+        return super(CrmTeam, self)._graph_title_and_key()
+
+    def _compute_dashboard_button_name(self):
+        no_opportunity_teams = self
+        if hasattr(self, 'use_opportunities'):
```

Comment:
Remove this `hasattr`.
Create `crm_team.py` in sale_crm module (bridge auto install, already existing), and move that code there.
Also, in sale_crm module, remove the dependendy to 'web_kanban_gauge'.